### DISCUSSION TITLE
Importing (some) models from openwebui.com fails

### DISCUSSION THREAD

**Discussion by jmtatsch** [open-webui/open-webui#1058] [General]
# Bug Report

## Description

**Bug Summary:**
Cant download the phi-2-dolphin model. Some others work.

**Steps to Reproduce:**
I go to
https://openwebui.com/m/c00100011/microsoft-phi-2-dolphin:latest
click get -> import webui -> referred to my local webui -> save & create -> error pull model manifest: file does not exist
click get -> download as a file -> file downloads but has .txt ending and thus is not shown in the file open dialog
one I rename the file to json it shows but still doesn't import as obviously the format is not real json

**Expected Behavior:**
I expect the model to be created and download the necessary files

**Actual Behavior:**
nothing, not even a useful error message

## Environment

- **Operating System:** 
- macOS Sonoma
- **Browser (if applicable):**
-  Safari

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I have reviewed the troubleshooting.md document.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

Docker compose

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Unable to reproduce the issue, could you make sure you're on the latest version?

**Comment by jmtatsch** [comment]
Just upgraded to v0.1.108. Problem is still there.
The other phi recipe works https://openwebui.com/m/sukmyfinga/sme-dolphin-phi:latest 
So maybe it's just an older recipe that has become incompatible with updates?

**Comment by nathanhere** [comment]
First time user, also getting this error.

Logs: 
```
INFO:   - "POST /api/v1/modelfiles/ HTTP/1.1" 401 Unauthorized
INFO:   - "POST /ollama/api/api/create HTTP/1.1" 500 Internal Server Error
```
is that "api/api" an accident?

**Comment by justinh-rahb** [comment]
> is that "api/api" an accident?

Nope, long story but we'll just say _backwards compatibility_.

**Comment by nathanhere** [comment]
ok, more info that might be helpful: it looks like using docker compose, this problem goes away. Previously i was using the "docker run" command when I experienced this issue.

Running on latest preview build of Windows 11, latest version of docker desktop.

**Comment by justinh-rahb** [comment]
> ok, more info that might be helpful: it looks like using docker compose, this problem goes away. Previously i was using the "docker run" command when I experienced this issue.

Despite seeming unrelated at first glance, there must be a discernible difference in how Docker is being configured or utilized between your `docker run` command and what Docker Compose generates. This difference is likely the key to understanding why the problem resolves when using Docker Compose. Check what you've got in `docker image ls`, this might reveal something.

If the solution with Docker Compose is working to your satisfaction, there might not be an immediate need to investigate further. However, it's important to note that there is still a possibility, even if small, that this issue—or another issue related to the difference between the `docker run` command and Docker Compose's approach—could resurface and cause problems in the future.

**Comment by jmtatsch** [comment]
I run using docker compose and have that issue.
But I think it's an issue with that model file as some other models work as expected.

**Comment by tjbck** [comment]
Might have something to do with your Ollama version, make sure you're on the latest version of Ollama as well.

**Comment by jmtatsch** [comment]
Looked a  bit deeper and turns out there are actually two issues here:

1. This recipe is just broken https://openwebui.com/m/c00100011/microsoft-phi-2-dolphin:latest - it looks for dolphin-2_6-phi-2.Q6_K.gguf:latest likely on ollama library which simply doesn't exist (anymore?) Thus the no manifest error.
2. Import of Modelfiles is broken. Likely the filetype filter is set up for json instead of txt.
![image](https://github.com/open-webui/open-webui/assets/3080674/f5fc86bd-5e76-4363-8dd6-f32405be55f2)

My ollama version is up to date.

**Comment by pxl-src** [comment]
Got the same behavior on Mac OS with latest safari but it's OK with google chrome

local ollama for the GPU

docker webui 
Open WebUI Version
v0.1.117
[(latest)](https://github.com/open-webui/open-webui/releases/tag/v0.1.117)

Voir les nouveautés
Vérifier les mises à jour
Version Ollama
0.1.30

any import to localhost:3000 finish in a 404 internal error in safari 
docker log 
open-webui  | INFO:     192.168.65.1:64286 - "GET /manifest.json HTTP/1.1" 200 OK
open-webui  | INFO:     192.168.65.1:64287 - "GET /_app/version.json HTTP/1.1" 304 Not Modified
open-webui  | INFO:     192.168.65.1:64286 - "GET /api/config HTTP/1.1" 200 OK
open-webui  | INFO:     192.168.65.1:64286 - "GET /api/v1/auths/ HTTP/1.1" 200 OK

docker compose 

version: '3.9'

services:
  open-webui:
    container_name: open-webui
    image: ghcr.io/open-webui/open-webui:main
    ports:
      - 3000:8080
    environment:
      - 'WEBUI_SECRET_KEY=###########'
    volumes:
      - ./open-webui:/app/backend/data

    extra_hosts:
     - host.docker.internal:host-gateway
    restart: unless-stopped
